packages/sanity/src/structure/panes/document/documentPanel/banners/DraftLiveEditBanner.tsx

```diff
@@ -0,0 +1,74 @@
+import {ErrorOutlineIcon} from '@sanity/icons'
+import {useTelemetry} from '@sanity/telemetry/react'
+import {Flex, Text} from '@sanity/ui'
+import {useCallback, useEffect, useState} from 'react'
+import {isDraftId, useDocumentOperation, useTranslation} from 'sanity'
+
+import {Button} from '../../../../../ui-components'
+import {structureLocaleNamespace} from '../../../../i18n'
+import {useDocumentPane} from '../../useDocumentPane'
+import {
+  DiscardedLiveEditDraft,
+  PublishedLiveEditDraft,
+} from './__telemetry__/DraftLiveEditBanner.telemetry'
+import {Banner} from './Banner'
+
+export function DraftLiveEditBanner(): JSX.Element | null {
```

Comment:
for consistency: I think we tend to not specify return type for react components - although I see there's a lint warning here.

Reply:
I tend to write the return types on every components I write exactly because of the lint waring 😬 
should I remove it?

Reply:
let's leave it as-is for now and discuss internally whether we want to keep the lint rule or not :)

Reply:
I would prefer to keep these, unless we make an active decision to not add the return types. 
If we have a linting rule then we should follow that rule otherwise we should remove it :)